pkg/lib/cockpit.d.ts

```diff
@@ -212,12 +214,12 @@ declare module 'cockpit' {
     function ngettext(context: string, message1: string, messageN: string, n: number): string;
 
     function format(format_string: string, ...args: unknown[]): string;
-    function format_number(n: number, precision?: number): string
-    function format_bytes(n: number, factor?: 1000 | 1024, options?: FormatOptions & { separate?: false }): string;
-    function format_bytes(n: number, factor: 1000 | 1024, options: FormatOptions & { separate: true }): string[];
-    function format_bytes_per_sec(n: number, factor?: 1000 | 1024, options?: FormatOptions & { separate?: false }): string;
-    function format_bytes_per_sec(n: number, factor: 1000 | 1024, options: FormatOptions & { separate: true }): string[];
-    function format_bits_per_sec(n: number, factor?: 1000 | 1024, options?: FormatOptions & { separate?: false }): string;
-    function format_bits_per_sec(n: number, factor: 1000 | 1024, options: FormatOptions & { separate: true }): string[];
-    function get_byte_units(guide_value: number, factor?: 1000 | 1024): ByteUnit[];
+    function format_number(n: number | null | undefined, precision?: number): string
+    function format_bytes(n: number | null | undefined, factor?: 1000 | 1024 | string, options?: FormatOptions & { separate?: false } | false): string;
+    function format_bytes(n: number | null | undefined, factor: 1000 | 1024 | string | undefined, options: FormatOptions & { separate: true } | true): string[];
+    function format_bytes_per_sec(n: number, factor?: 1000 | 1024 | string, options?: FormatOptions & { separate?: false } | false): string;
+    function format_bytes_per_sec(n: number, factor: 1000 | 1024 | string | undefined, options: FormatOptions & { separate: true } | true): string[];
+    function format_bits_per_sec(n: number, factor?: 1000 | 1024 | string | undefined, options?: FormatOptions & { separate?: false } | false): string;
+    function format_bits_per_sec(n: number, factor: 1000 | 1024 | string | undefined, options: FormatOptions & { separate: true } | true): string[];
+    function get_byte_units(guide_value: number, factor?: 1000 | 1024): [ByteUnit, ByteUnit, ByteUnit];
```

Comment:
FTR: There are *zero* uses of this outside of cockpit.git, and it's not documented. The only place that uses it is in cockpit/main/pkg/storaged/dialog.jsx . Let's make that private?